rfcs/react-components/components/combobox-spacebar.md

```diff
@@ -0,0 +1,115 @@
+
+## Precedence
+
+- In v8 Combobox and v0 Dropdown, spacebar did not select
+- Office editable comboboxes select on space when opened with alt+down, but not when opened with the down arrow alone
+- On Windows, non-editable comboboxes (similar to v9 Dropdown) in Settings select with space. Editable comboboxes in apps like Explorer and Activity monitor allow space as a character
+
+## User study
+
+In an A/B user study focusing primarily on screen reader users and alternative input users (keyboard, switch, voice control), people using the keyboard overwhelmingly expected space to select in all comboboxes. In variations where space did not select, this was called out as one of the primary confusing things about the component, particularly strongly with multiselect comboboxes.
+
+The two caveats to consider when evaluating the user study are that it is impossible to fully and realistically recreate real-world app context and expectations in an isolated test UI, and also that it focused on a small group (~12) of participants who all used some form of assistive tech. It points towards an expected behavior, but is not an absolute conclusion.
+
+## Context
+
+The surrounding context of an app or website is also likely to influence user expectations on keyboard behavior. Typing a name into a `To: ` input in Outlook or Teams is likely to influence users to expect space to insert a character. Moving through multiple fields in a form is more likely to influence users to expect space to select. The ideal spacebar behavior is likely to differ based on how and where the v9 Combobox is being used.
+
+The content of options also influences user expectations -- there would be a greater expectation of space-to-type when options contain a space in them, for example multi-word options, names, states or cities.
+
+One of the risks is that the developers and designers may not be in a position to make a correct decision on the value of the prop. This is because options are usually either translations or user data, which may contain spaces that the authoring team was not aware of. We generally can't expect authors to understand all possible translations across supported languages or the specific validations of the user data at UI authoring time. While the occaisonal space character does not necessarily mean that space should not select, translations and user data do introduce the possibility of frequent spaces within multiple options.
+
+## Problem statement
+
+We cannot fully know whether the ideal spacebar behavior is to select or insert a character, since we don't know the full context of usage. Our choices mostly revolve around balancing providing more flexibility vs. more consistency, and choosing the best default if we do provide flexibility.
+
+## Options
+
+### 1. `allowSpaceCharacter` prop, default to `true`
```

Comment:
In the context of HVCs, option 1 seems like it could cause significant confusion across apps, unless HVC authors explicitly allow some mechanism to override that prop. It would be hard for us to guarantee that across all layers of HVCs